generators/premiums/google_search/README.md

```diff
@@ -0,0 +1 @@
+Uses Google engine to retrieve search results. This module requires an API key to use. In refinery, set YOUR_RESPONSE_SIZE to "full" to get the full response as a JSON (stored as string) or choose "compact" to only get the text of the first result. Please contact the Kern AI Team if you need further support at info@kern.ai or join our Discord channel: https://discord.gg/WAnAgQEv 
```

Comment:
Could you add that this will only work with refinery > 1.7.1? Otherwise looks good to me. :-)

Reply:
Yep, I have added it :)